packages/block-editor/src/components/preview-options/README.md

```diff
@@ -0,0 +1,63 @@
+# Preview Options
+
+The `PreviewOptions` component displays the list of different preview options available in the editor.
+
+It returns a [`DropdownMenu`](https://github.com/WordPress/gutenberg/tree/master/packages/components/src/dropdown-menu) component with these different options. The options currently available in the editor are Desktop, Mobile, Tablet and "Preview in new tab".
+
+![Preview options dropdown menu](https://make.wordpress.org/core/files/2020/09/preview-options-dropdown-menu.png)
+
+## Table of contents
+
+1. [Development guidelines](#development-guidelines)
+2. [Related components](#related-components)
+
+## Development guidelines
+
+### Usage
+
+Renders the previews options of the editor in a dropdown menu.
+
+```jsx
+import { PreviewOptions } from '@wordpress/block-editor';
+
+const MyPreviewOptions = () => (
+    <PreviewOptions
+        isEnabled={ true }
+        className="edit-post-post-preview-dropdown"
+        deviceType={ deviceType }
+        setDeviceType={ setPreviewDeviceType }
+    >
+);
```

Comment:
Given that this component accepts `children`, should we have an example documenting that? We have [one instance](https://github.com/WordPress/gutenberg/blob/3e57ffdb824b0a313cfbebf812db5ce1bb068d2a/packages/edit-post/src/components/device-preview/index.js#L33) of its use in that format so far.

Reply:
Hum 🤔, I guess we can do that. For the past docs PRs I sent, I tried to keep it short and only add examples for the component itself.

Reply:
Are you happy to do this? Not a blocker, more of a nice-to-have 🙂

Reply:
Sure thing.
I can do something like

```jsx
import { Icon, MenuGroup } from '@wordpress/components';
import { PostPreviewButton } from '@wordpress/editor';
import { __experimentalPreviewOptions as PreviewOptions } from '@wordpress/block-editor';

const MyPreviewOptions = () => (
    <PreviewOptions
        isEnabled={ true }
        className="edit-post-post-preview-dropdown"
        deviceType={ deviceType }
        setDeviceType={ setPreviewDeviceType }
        >
        <MenuGroup>
            <div className="edit-post-header-preview__grouping-external">
                <PostPreviewButton
                    className={
                        'edit-post-header-preview__button-external'
                    }
                    role="menuitem"
                    forceIsAutosaveable={ hasActiveMetaboxes }
                    forcePreviewLink={ isSaving ? null : undefined }
                    textContent={
                        <>
                            { __( 'Preview in new tab' ) }
                            <Icon icon={ external } />
                        </>
                    }
                />
            </div>
		</MenuGroup>
    >
);
```

using the `DevicePreview` example.
Looks good?

Reply:
Sure, that works! One thing: the snippet above is missing the closing `</PreviewOptions>` tag; should add it in for clarity.